src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,93 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.command.Command;
+import duke.command.CommandKeyword;
+import duke.exception.DukeException;
+import duke.exception.InvalidCommandException;
+import duke.task.Task;
+import duke.task.TaskList;
+
+/**
+ * Deals with making sense of the user command and executing it.
+ */
+public class Parser {
+
+    /**
+     * Execute user command and returns a boolean that represents the continuation of the program.
```

Comment:
Should the comment start with "Executes" instead of "Execute"? I noticed this in several other places too.
```suggestion
     * Executes user command and returns a boolean that represents the continuation of the program.
```